.github/workflows/ci.yml

```diff
@@ -87,3 +87,36 @@ jobs:
       - run: cargo run -- --help
       - run: cargo run -- list-remote node
       - run: cargo run -- list-remote wasm-test
+  build:
+    name: Upload build
+    runs-on: ${{ matrix.host }}
+    strategy:
+      matrix:
+        include:
+        - target: aarch64-unknown-linux-gnu
+          host: ubuntu-latest
```

Comment:
youll need the build jet runner for this to work

Reply:
We should be able to cross-compile for aarch64 from x64.

I *did* need to install a suitable compiler with `sudo apt-get install gcc-aarch64-linux-gnu` and then it worked: https://github.com/rotu/proto/actions/runs/7279006762/job/19834420423?pr=1